doc/zephyr.doxyfile.in

```diff
@@ -279,6 +279,9 @@ ALIASES                = "rst=\verbatim embed:rst:leading-asterisk" \
                          "req{1}=\ref ZEPH_\1 \"ZEPH-\1\"" \
                          "satisfy{1}=\xrefitem satisfy \"Satisfies requirement\" \"Requirement Implementation\" \1" \
                          "verify{1}=\xrefitem verify \"Verifies requirement\" \"Requirement Verification\" \1" \
+                         "kconfig_dep{1}=\attention Available only when the follow Kconfig option is enabled: \kconfig{\1}." \
```

Comment:
Can't believe no one spotted the `s/follow/following/` typo. Will address once @Thalley has chimed in and will also remove the proof of concept commits

Reply:
The `s/follow/following/` typo also exist for the follow ( :wink: ) 2 lines

Reply:
yep I know, but thanks! :)